resources/linux/firewall/fedora_systemctl_status_firewalld.service_OFF.txt

```diff
@@ -0,0 +1,11 @@
+● firewalld.service - firewalld - dynamic firewall daemon
+   Loaded: loaded (/usr/lib/systemd/system/firewalld.service; disabled; vendor preset: enabled)
+   Active: active (running) since Tue 2017-03-14 15:33:24 CET; 1h 34min ago
```

Comment:
Weird, is this really the output of disabled firewalld service ? It looks enabled for me as "active and running"
The service is disabled okay, but the firewall is active at that moment.
I think regex is too strict and should not match only if service is enabled, but if it is active and running. This is my opinion.